src/Leaves/Registration/RegistrationView.php

```diff
@@ -44,6 +44,10 @@ protected function createSubLeaves()
                 $this->model->createUserEvent->raise();
             })
         );
+
+        if ($this->model->email != "") {
+            $this->leaves["email"]->setValue($this->model->email);
```

Comment:
Never call setValue directly if you can avoid it - this is called by the data binding bit of Leaf and can cause issues - what's curious here is that instead you should set a model property of the same name on the RegistrationModel but I see you must have already done that - so we should maybe try to understand why the email leaf isn't binding to that value properly.